recipes-core/images/aos-image-weston.bb

```diff
@@ -0,0 +1,10 @@
+SUMMARY = "A very basic Wayland image with AOS components"
+
+require aos-image-minimal.bb
+
+IMAGE_INSTALL += "wayland \
+                  weston \
```

Comment:
This approach doesn't seems to be the correct way to add weston.
Please see the link for details:https://wiki.yoctoproject.org/wiki/Wayland